### PR TITLE
fix(ui): adjust links

### LINKED ISSUE #1078: Component Documentation - fix links
### Description / User story

Currently it seems like all links to other component docs pages are broken:

eg: In https://b01ler.onrender.com/?path=/docs/components-checkbox--docs#form-label if you click the link (Form Label) in For more information have a look at the internal [Form Label](https://b01ler.onrender.com/?path=/docs/design-system-web-components-internal-components-formlabel--docs) component. 

The redirect does not work. It is also happening for other components as well.

### Requirements / Prerequisites

- none

### Acceptance Criteria

- [ ] All links to other component docs pages work again

### Additional information

- ...

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by bar-tay**
closes #1078

### CODE REVIEW

File: packages/ui-library/src/components/form-caption/index.stories.ts
Comment (thrbnhrtmnn) [inline packages/ui-library/src/components/form-caption/index.stories.ts:1]: This issue already existed before your PR, but we should fix it quickly within this PR: In the Intro the Table of Content and the List of components that make use of the Form Caption is badly formatted so both sections merged together. Could you add a break before "The Form Caption is intended to be used when creating new components. Currently, it is used like this in the following components:"?

File: packages/ui-library/src/components/input-field-number/index.stories.ts [hunk 1]
```diff
@@ -607,7 +607,7 @@ export const HasError = (params: BlrInputFieldNumberType) => {
 /**
  * ## Dependencies
  * ### Button Icon
- * The Input Field Number component makes use of the Button Icon component for increasing or decreasing the value. For more information have a look at the [Button Icon](?path=/docs/design-system-web-components-actions-buttons-button-icon--docs) component.
+ * The Input Field Number component makes use of the Button Icon component for increasing or decreasing the value. For more information have a look at the [Button Icon](?path=/docs/components-actions-buttons-button-icon--docs) component.
```
Comment (thrbnhrtmnn) [inline packages/ui-library/src/components/input-field-number/index.stories.ts:610]: The Button Icon link is not working

File: packages/ui-library/src/components/input-field-text/index.stories.ts [hunk 2]
```diff
@@ -440,7 +440,7 @@ SizeVariant.story = {
 /**
  * ## Content / Settings
  * ### Type
- * The Input Field Text component can have all the types an html input can have, except the number and the unit type, which is covered in the Input Field Number component. For more information have a look at the [Input Field Number](/docs/design-system-web-components-forms-input-field-number--docs) component.
+ * The Input Field Text component can have all the types an html input can have, except the number and the unit type, which is covered in the Input Field Number component. For more information have a look at the [Input Field Number](/docs/components-forms-input-field-number--docs) component.
```
Comment (thrbnhrtmnn) [inline packages/ui-library/src/components/input-field-text/index.stories.ts:443]: The Input Field Number link is not working

File: packages/ui-library/src/components/toggle-switch/index.stories.ts [hunk 3]
```diff
@@ -413,7 +413,7 @@ Readonly.story = { name: ' ' };
 /**
  * ## Dependencies
  * ### Form Caption
- * The Toggle Switch component can display an optional hint message with or without icons. For more information have a look at the [Form Caption](/docs/design-system-web-components-internal-components-formcaptiongroup-formcaption--docs) component.
+ * The Toggle Switch component can display an optional hint message with or without icons. For more information have a look at the [Form Caption](/docs/components-form-caption-group-formcaption--docs) component.
```
Comment (thrbnhrtmnn) [inline packages/ui-library/src/components/toggle-switch/index.stories.ts:416]: Link to the Form Caption is not working (it should not link to form caption group)

**Review by thrbnhrtmnn — CHANGES_REQUESTED** [review]

File: packages/ui-library/src/components/input-field-number/index.stories.ts (hunk 1 above)
Comment (bar-tay) [reply packages/ui-library/src/components/input-field-number/index.stories.ts:610]: done

File: packages/ui-library/src/components/input-field-text/index.stories.ts (hunk 2 above)
Comment (bar-tay) [reply packages/ui-library/src/components/input-field-text/index.stories.ts:443]: done

File: packages/ui-library/src/components/input-field-number/index.stories.ts (hunk 1 above)
Comment (thrbnhrtmnn) [reply packages/ui-library/src/components/input-field-number/index.stories.ts]: Sorry @bar-tay , the whole Button Icon section was removed with #1015 so this review was not up to date anymore. I see that you added the section back to the documentation, this is however not needed anymore. Can you remove it again?

File: packages/ui-library/src/components/form-caption/index.stories.ts
Comment (bar-tay) [reply packages/ui-library/src/components/form-caption/index.stories.ts:1]: done

File: packages/ui-library/src/components/input-field-text/index.stories.ts (hunk 2 above)
Comment (thrbnhrtmnn) [reply packages/ui-library/src/components/input-field-text/index.stories.ts:443]: Link still not working.

I think the `-forms` in the link needs to be removed as well. The links should be `/docs/components-input-field-number--docs`, if I am not mistaken.

File: packages/ui-library/src/components/toggle-switch/index.stories.ts (hunk 3 above)
Comment (bar-tay) [reply packages/ui-library/src/components/toggle-switch/index.stories.ts:416]: done

File: packages/ui-library/src/components/input-field-text/index.stories.ts (hunk 2 above)
Comment (bar-tay) [reply packages/ui-library/src/components/input-field-text/index.stories.ts:443]: done

File: packages/ui-library/src/components/input-field-number/index.stories.ts (hunk 1 above)
Comment (bar-tay) [reply packages/ui-library/src/components/input-field-number/index.stories.ts]: done

**Review by thrbnhrtmnn — APPROVED** [review]
Looks good in Storybook! Thank you @bar-tay

File: packages/ui-library/src/components/form-caption/index.stories.ts [hunk 4]
```diff
@@ -73,16 +73,19 @@ export default {
       description: {
         component: `<Markdown>
         Form Caption provides either a brief hint or error message to a related form element such as Input Field Text or Select. It is typically displayed below the form element, and it can be used to provide additional information about the type of information a user has to provide, like an example of the correct data format.
+
```
Comment (angsherpa456) [inline packages/ui-library/src/components/form-caption/index.stories.ts:76]: Please unify these new lines for consistency. Right now some of them has new line and some have missing. Same for line 82 and 88.

File: packages/ui-library/src/components/input-field-number/index.stories.ts [hunk 5]
```diff
@@ -607,6 +607,7 @@ export const HasError = (params: BlrInputFieldNumberType) => {
 
 /**
  * ## Dependencies
+
```
Comment (angsherpa456) [inline packages/ui-library/src/components/input-field-number/index.stories.ts:610]: Please unify these new lines for consistency.

File: packages/ui-library/src/components/form-caption/index.stories.ts (hunk 4 above)
Comment (bar-tay) [reply packages/ui-library/src/components/form-caption/index.stories.ts:76]: done

File: packages/ui-library/src/components/input-field-number/index.stories.ts (hunk 5 above)
Comment (bar-tay) [reply packages/ui-library/src/components/input-field-number/index.stories.ts:610]: done

**Review by angsherpa456 — APPROVED** [review]
LGTM

**Pull request merged**